README.rst

```diff
@@ -41,6 +41,29 @@ If a Python core developer approved a PR made by anyone and added the "🤖 auto
 it will be automatically merged once all the CI checks pass.
 
 
+Setup Info
+==========
+
+Requires Python 3.6+
+
+```
+$ python3 -m venv venv
+$ source venv/bin/activate
+
+// for development
```

Comment:
Instead of C-like comments, it might be better to use plain text and separate sections like:

Create the venv and activate it:
```sh
$ python3 -m venv venv
$ source venv/bin/activate
```

Install development or production dependencies:
```sh
(venv) $ pip install -r dev-requirements.txt  # for development
(venv) $ pip install -r requirements.txt      # for production
```

Run the tests with:
```sh
(venv) $ pytest -v
```